src/svgcontext.ts

```diff
@@ -458,12 +458,8 @@ export class SVGContext implements RenderContext {
   }
 
   fillRect(x: number, y: number, width: number, height: number): this {
-    if (height < 0) {
-      y += height;
-      height *= -1;
-    }
```

Comment:
Do we know what this was supposed to accomplish? Is there a test case for negative heights?

Supposedly the Canvas API supports negative widths and heights. Not sure how closely we are trying to replicate the Canvas API:

https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/fillRect

Reply:
SVG doesn't support negative widths or heights, so this code was correcting for that.

However, `fillRect` calls through to `rect`, which also performs the same correction so this code is entirely redundant.

Reply:
Ah makes sense. The default github diff only showed this snippet, and I neglected to expand the view to see the implementation of rect(). I suppose we don't need to handle negative widths yet. :-)